src/__stories__/main-navigation-bar-story.tsx

```diff
@@ -29,6 +30,16 @@ export const Default: StoryComponent<Args> = ({isInverse}) => {
             isInverse={isInverse}
             sections={sections.map((title, idx) => ({title, onPress: () => setSelectedIndex(idx)}))}
             selectedIndex={selectedIndex}
+            // todo WEB-761 more logo controls, like logoType? No explicit logo sample?
+            // logo={
+            //     <Logo
+            //         logoType="isotipo"
+            //         onPress={() => {
+            //             alert('pressed');
+            //         }}
+            //         size={48}
+            //     />
+            // }
```

Comment:
I've left like this in order to let the old tests pass without any breaking change, but maybe it's more explanatory for future users with this? you know, at a first glance you're aware of this by putting it explicitly